src/main/resources/com/ncc/aif/ontologies/InterchangeOntology

```diff
@@ -160,6 +160,18 @@
   rdfs:range xsd:float ;
   rdfs:subPropertyOf owl:topDataProperty ; .
 
+:importance rdf:type owl:DatatypeProperty ;
+  rdfs:comment "Value for importance represented as a double." ;
+  rdfs:domain [
+        rdf:type owl:Class ;
+        owl:unionOf (
+            :rdf:Statement
+            :Hypothesis
+            :SameAsCluster ) ] ;
+  rdfs:label "importance" ;
+  rdfs:range xsd:double ;
```

Comment:
I know we're not supporting the GAIA version, but since we added importance to its version of InterchangeOntology, should we perhaps update its type to a double, or just remove it?

Reply:
Type updated